src/shared/components/annotation/genomeNexus/PolyPhen2.tsx

```diff
@@ -22,7 +22,7 @@ export default class PolyPhen2 extends React.Component<IPolyPhen2Props, {}> {
         this.tooltipContent = this.tooltipContent.bind(this);
     }
 
-    public static download(polyPhenScore: number, polyPhenPrediction: string): string
+    public static download(polyPhenScore: number | null, polyPhenPrediction: string | null): string
```

Comment:
what happens if these are null?

Reply:
i believe that we would actually get the the text "null" printed

Reply:
i believe it would be "null". actually when I test it I found the functional impact download function was not working, some of my changes might break the function, I'll fix it soon

Reply:
i fix it to show "NA" when there is no data.